homeassistant/components/opple/manifest.json

```diff
@@ -5,5 +5,5 @@
   "documentation": "https://www.home-assistant.io/integrations/opple",
   "iot_class": "local_polling",
   "loggers": ["pyoppleio"],
-  "requirements": ["pyoppleio==1.0.5"]
+  "requirements": ["pyoppleio-310==1.0.8"]
```

Comment:
We don't switch libraries as a quick fix. Have you tried asking the maintainer of the original library to make you a maintainer or transferring the project to you?

If you're willing to maintain the library and have confirmed that the maintainer of the original library is unresponsive for a long period of time (eg longer than a month), you should create a new library with a different name (just adding `-310` is not ok). It can be a copy of the fork, but it should not use the original library as upstream.

Reply:
ok, I'll try get in touch with the origin maintainer first.

Reply:
大佬有没有打算建一个新库呀🤥我每周都在看opple的问题有没有被修复，太累了。

Reply:
Please post in English.